app/main.py

```diff
@@ -1,3 +1,49 @@
+import math
+
+
 class Vector:
-    # write your code here
-    pass
+
+    def __init__(self, x: float = 0.00, y: float = 0.00):
+        self.x = round(x, 2)
+        self.y = round(y, 2)
+
+    def __add__(self, other):
+        return Vector(x=round(self.x, 2) + round(other.x, 2),
+                      y=round(self.y, 2) + round(other.y, 2))
+
+    def __sub__(self, other):
+        return Vector(x=self.x - round(other.x, 2),
+                      y=self.y - round(other.y, 2))
+
+    def __mul__(self, other):
+        if isinstance(other, Vector):
+            return (self.x * other.x) + (self.y * other.y)
+        return Vector(x=round(self.x * other, 2), y=round(self.y * other, 2))
+
+    @staticmethod
+    def create_vector_by_two_points(start_point: tuple, end_point: tuple):
+        return Vector(round(end_point[0] - start_point[0], 2),
+                      round(end_point[1] - start_point[1], 2))
+
+    def get_length(self):
+        return (self.x ** 2 + self.y ** 2) ** 0.5
+
+    def get_normalized(self):
+        return Vector(x=round(self.x / (self.x ** 2 + self.y ** 2) ** 0.5, 2),
+                      y=round(self.y / (self.x ** 2 + self.y ** 2) ** 0.5, 2))
+
+    def angle_between(self, other):
+        cos_a = (self.x * other.x + self.y * other.y) / (
```

Comment:
try to write `self + other`

Reply:
(self.x * other.x + self.y * other.y)   Essentially this part of the expression can be written as a method __mul__ from self and other. I do not know how correctly write the syntax

Reply:
<img width="442" alt="image" src="https://user-images.githubusercontent.com/72562232/176370010-e6f2e8e1-972e-4f25-9e3a-e5c896f5984c.png">
why do you need this methid?)